### PR TITLE
doc: openthread: remove `overlay-cert.conf`

### PR DESCRIPTION
**Pull request by edmont**
Remove Thread Harness files in favor of simplicity.

Signed-off-by: Eduardo Montoya <eduardo.montoya@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/ug_thread_certification.rst [hunk 1]
```diff
@@ -64,19 +64,24 @@ Complete the following steps to prepare for the certification tests:
 
 #. Build the certification image.
 
-   The :ref:`ot_cli_sample` sample is used as a base, modified with the :file:`harness/overlay-cert.conf` and :file:`overlay-multiprotocol.conf` overlay files.
+   The :ref:`ot_cli_sample` sample is used as a base, modified with the :file:`overlay-ci.conf` and :file:`overlay-multiprotocol.conf` overlay files.
```
Comment (wiba-nordic) [inline doc/nrf/ug_thread_certification.rst:67]:
```suggestion
   Use the :ref:`ot_cli_sample` sample as a base, and apply the :file:`overlay-ci.conf` and :file:`overlay-multiprotocol.conf` overlay files.
```
Better to change this to imperative as well.

File: doc/nrf/ug_thread_certification.rst [hunk 2]
```diff
@@ -64,19 +64,24 @@ Complete the following steps to prepare for the certification tests:
...
 
      .. code-block::
 
         cd ncs/nrf/samples/openthread/cli/
-        west build -b nrf52840dk_nrf52840 -- -DOVERLAY_CONFIG="harness/overlay-cert.conf;overlay-multiprotocol.conf" -DCONFIG_OPENTHREAD_LIBRARY=y
+        west build -b nrf52840dk_nrf52840 -- -DOVERLAY_CONFIG="overlay-ci.conf;overlay-multiprotocol.conf" -DCONFIG_OPENTHREAD_LIBRARY=y -DCONFIG_LOG=n -DCONFIG_ASSERT=n
 
    * If building using Visual Studio Code, you must first `create the application <Creating an application_>`_ using the CLI sample, and then `build the application <Building an application_>`_.
-     Select the :file:`harness/overlay-cert.conf` and :file:`overlay-multiprotocol.conf` overlay files in the :guilabel:`Kconfig fragment` drop-down menu and add ``CONFIG_OPENTHREAD_LIBRARY=y`` to the **Additional CMake arguments** text field.
+     Select the :file:`overlay-ci.conf` and :file:`overlay-multiprotocol.conf` overlay files in the :guilabel:`Kconfig fragment` drop-down menu and add the following lines to the **Additional CMake arguments** text field.
```
Comment (wiba-nordic) [inline doc/nrf/ug_thread_certification.rst:78]:
```suggestion
     Select the :file:`overlay-ci.conf` and :file:`overlay-multiprotocol.conf` overlay files in the :guilabel:`Kconfig fragment` drop-down menu and add the following lines to the **Additional CMake arguments** text field:
     
```
Note the added empty line between the text and the code block for consistency.

File: doc/nrf/ug_thread_certification.rst [hunk 3]
```diff
@@ -89,7 +94,7 @@ Complete the following steps to prepare for the certification tests:
 
    a. Copy the provided :file:`ncs/modules/lib/openthread/tools/harness-thci/OpenThread.py` file into :file:`C:\\GRL\\Thread1.1\\Thread_Harness\\THCI\\nRF_Connect_SDK.py`.
 
-   b. Copy the provided :file:`ncs/nrf/samples/openthread/cli/harness/nRF_Connect_SDK.jpg` file into :file:`C:\\GRL\\Thread1.1\\Web\\images\\`.
+   b. Copy any image that you desire to :file:`C:\\GRL\\Thread1.1\\Web\\images\\nRF_Connect_SDK.jpg`.
```
Comment (wiba-nordic) [inline doc/nrf/ug_thread_certification.rst:97]:
```suggestion
   b. Copy an image of your choice to :file:`C:\\GRL\\Thread1.1\\Web\\images\\nRF_Connect_SDK.jpg`.
```

File: doc/nrf/ug_thread_ot_memory.rst [hunk 4]
```diff
@@ -20,8 +20,9 @@ The memory requirement tables list flash memory and RAM requirements for the :re
 The values change depending on device type and hardware platform.
 Moreover, take into account the following considerations:
 
-* The sample was compiled using the default :file:`prj.conf` plus :file:`harness\overlay-cert.conf`  and :file:`overlay-multiprotocol.conf` overlay files, and the corresponding :kconfig:option:`OPENTHREAD_NORDIC_LIBRARY`, :kconfig:option:`OPENTHREAD_THREAD_VERSION` choices.
-* The single protocol sample was configured by setting :kconfig:option:`CONFIG_BT` to ``n``.
+* The sample was compiled using the default :file:`prj.conf` and the corresponding :kconfig:option:`OPENTHREAD_NORDIC_LIBRARY`, :kconfig:option:`OPENTHREAD_THREAD_VERSION` choices.
+* The sample was configured by setting :kconfig:option:`CONFIG_ASSERT` to ``n``.
```
Comment (wiba-nordic) [inline doc/nrf/ug_thread_ot_memory.rst:24]:
```suggestion
* The sample was compiled using:
   
   * The default :file:`prj.conf` and the corresponding :kconfig:option:`OPENTHREAD_NORDIC_LIBRARY`, :kconfig:option:`OPENTHREAD_THREAD_VERSION` choices.
   * The :kconfig:option:`CONFIG_ASSERT` Kconfig option set to ``n``.

```
This doesn't use the CONFIG_LOG set to `n` that was used for certification?

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

File: doc/nrf/ug_thread_ot_memory.rst (hunk 4 above)
Comment (edmont) [reply doc/nrf/ug_thread_ot_memory.rst]: Not currently. Maybe we could think about unifying configurations.

**Review by wiba-nordic — APPROVED** [review]

**Review by canisLupus1313 — APPROVED** [review]

**Pull request merged**